### ISSUE TITLE
Tooltip box extends outside of viewport in bottom-most plot

### ISSUE THREAD

**Issue by themightyoarfish** [tensorflow/tensorboard#1607]
- TensorBoard version (from pip package, also printed out when running `tensorboard`) 1.12.0
 - TensorFlow version if different from TensorBoard: see above
 - OS Platform and version: Linux Ubuntu 16.04
 - Python version: 3.6.5
 - For browser-related issues:
   - Browser type and version: Firefox 63.0.1 on macOS (tb running remotely)
   - Screenshot if it's a visual issue

![screen shot 2018-11-12 at 10 42 54](https://user-images.githubusercontent.com/11613312/48339218-f9b1d000-e667-11e8-88c5-f7120622bae1.png)

For the bottom plot, the tooltip box moves outside the viewport, so nothing can be read.
I'm sure this has been raised before, but I can't seem to find the issue.

**Comment by stephanwlee** [comment]
Hi @themightyoarfish, thanks for raising the issue. Yup, it is a known issue and is similar to #1170. 
What do you think we should do in this case? From what I can see, the tooltip is too large to place it above the chart. Also because the tooltip does not let you put the cursor, you can't scroll there. I am genuinely interested to hear a possible solution to this. Thanks!

**Comment by themightyoarfish** [comment]
I think the most straightforward solution would be to extend the page some below the last plot, so that at least I can scroll the plot all the way to the top of the page and have the tooltip be visible. Of course I couldn't see the entire plot and tooltip at the same time, but at least it would enable me to figure out which line is which by finding the color in the tooltip. In such a case, the only alternative would be toggling lines in the selector and see which one disappears.

**Comment by stephanwlee** [comment]
What kind of screen dimension do you have? An easy workaround would be to zoom out and make everything appear tiny which really isn't ideal.

**Comment by themightyoarfish** [comment]
The screen is 1280x800. Zooming out is sometimes possible, yes. But ofc the usefulness is limited by the screen resolution.

**Comment by nfelt** [comment]
I think this could be addressed by #1019 - the motivation there was to be able to see very long tooltips (longer than visible in a single viewport), but I think it also applies to this case where the tooltip extends past the bottom of the scrollable page.

The basic idea in #1019 was to make the tooltip somehow "pinnable" so that it doesn't disappear when you try to move the cursor over it, scroll, etc.  This would allow interacting with the tooltip as a "touchable" page component, which unblocks things like selecting and copying text from it, potentially sorting, etc.

Still TBD was whether the pinning action should require some extra action (like clicking a button, or even triggering the tooltip in some other way like via a button below the chart), or whether it should be the default behavior if you move the hovering cursor over the tooltip (it becomes resident and "touchable" until you either move the cursor off it, or perhaps even have to manually dismiss it via an "X" button).

**Comment by nfelt** [comment]
Duplicate of #1019
